lib/tasks/browserstack.rake

```diff
@@ -41,11 +41,11 @@ namespace :browserstack do
         # We're in a subprocess here - set the environment variable BS_BROWSER to the desired browser configuration.
         ENV['BS_BROWSER'] = browser.to_json
 
-        test_label = ['os', 'os_version', 'browser', 'browser_version'].map { |k| browser[k] }.join('_')
+        test_label = ['os', 'os_version', 'browser', 'browser_version'].map(&:browser).join('_')
 
         system("rspec spec/features --format RspecJunitFormatter --out '#{test_label}.xml'")
       end
-      exitstatus = results.count { |e| !e }
+      exitstatus = results.count(&:'!e')
```

Comment:
@dan-palmer I'll take a look.  Is it worth bothering with at all if it's never worked and we plan to take it out soon anyway?